hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/constants"
```

Comment:
Imports should be grouped in a way consistent with [Effective Go style suggestion](https://github.com/golang/go/wiki/CodeReviewComments#imports) as well as the rest of the tanzu-framework repository. Here is [an example from featuregates sdk](https://github.com/vmware-tanzu/tanzu-framework/blob/b0ca18902d7da98a667322ab263701b0a1716f7b/pkg/v1/sdk/features/featuregate/featuregate.go#L13).

I suggest grouping imports as such, separated by a single empty line:
- standard library packages
- packages from external projects
- packages from the tanzu-framework repository

Reply:
Related to my comment earlier: https://github.com/vmware-tanzu/tanzu-framework/pull/1512#issuecomment-1058507435

Please fix running the linting on this code. There are a few things like this that it will point out for you.